CleanWeather/Helpers/AlertPresentable.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  AlertPresentable.swift
+//  CleanWeather
+//
+//  Created by Patryk Średziński on 19/11/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+import UIKit
+
+protocol AlertPresentable: UIViewController {
+    
+    func presentAlert(title: String, message: String)
+    
+}
+
+extension AlertPresentable {
+    
+    func presentAlert(title: String, message: String) {
+        let alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
+        let action = UIAlertAction(title: R.string.localizable.ok(), style: .default, handler: nil)
+        alert.addAction(action)
+        navigationController?.present(alert, animated: true, completion: nil)
```

Comment:
Nad tym też się zastanawiałem.
A co jeśli nie będziemy używać navigationController a np tabBarController? Czy tego też nie powinniśmy jakoś ujednolicić żeby można to było użyć w innej aplikacji z tabBarem :)

Reply:
raczej nasze kontrolery zawsze będą miały navigationbarcontroller
a jak będzie tabbar, to tabbar bęędzie miał w sobie navigationbarcontrollery
i powyższy kod działać będzie